CMakeLists.txt

```
@@ -65,6 +66,15 @@ target_link_libraries (AimIO
 		Boost::filesystem Boost::system
 )
 
+option (N88_BUILD_AIX "Build aix tool." ON)
+if (N88_BUILD_AIX)
+		add_executable (aix source/aix.cxx)
+		target_link_libraries (aix AimIO)
+		if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
+		    target_link_libraries (AimIO dl pthread)
```

I believe you have the wrong target here. Should it not be `aix`?
```suggestion
		    target_link_libraries (aix dl pthread)
```